services/keylime_agent.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Wants=keylime_verifier.service keylime_registrar.service
+After=network.target keylime_verifier.service keylime_registrar.service
+
+[Service]
+ExecStart=/usr/local/bin/keylime_agent
```

Comment:
Looks great and the install script worked well (thanks for your contribution too!)

The only thing I found was the path is different for me.

I am on Fedora 30 and used `python setup.py install` 

This resulted in:

```
Jul 18 14:16:19 localhost.localdomain systemd[4798]: keylime_verifier.service: Failed to execute command: No such file or directory
Jul 18 14:16:19 localhost.localdomain systemd[4798]: keylime_verifier.service: Failed at step EXEC spawning /usr/local/bin/keylime_verifier
```

I edited the service file and changed the path to `/usr/bin/keylime_verifier` and it works well.

I wonder if this is linux distribution specific or the `installer.sh`

Which Linux dist did you test this against, and did you use setuptools or the `installer.sh`? I think once we iron this out, the patch will be good to merge!

Reply:
> I can update the services installer by checking first where are located the keylime_agent, keylime_verifier and keylime_registrar on the target system and then update the service files accordingly with some sed or similar tools.

I think that's a good idea... have a condition statement to check the file exists and then set the path accordingly. I would be happy with that!